components/popover/popover-mixin.js

```diff
@@ -100,19 +106,41 @@ export const PopoverMixin = superclass => class extends superclass {
 			this._previousFocusableAncestor = this.opened ? getPreviousFocusableAncestor(this, false, false) : null;
 
 			if (this.opened) {
+
 				this._opener = getComposedActiveElement();
 				this._addAutoCloseHandlers();
-				this._dismissibleId = setDismissible(() => this._close());
+				this._dismissibleId = setDismissible(() => this.close());
+				this._focusContent(this);
```

Comment:
With `tooltip` I believe we don't generally move focus into the content. Would the expectation be that tooltip would be configured with `noAutoFocus`?

Reply:
I think so, yeah. This is also one of the reasons I am hoping to pursue the idea that Dave has to use an internal `configure()` method, so that these properties are not exposed on tooltip.